pkg/services/user/userimpl/user.go

```diff
@@ -209,14 +209,7 @@ func (s *Service) Delete(ctx context.Context, cmd *user.DeleteUserCommand) error
 }
 
 func (s *Service) GetByID(ctx context.Context, query *user.GetUserByIDQuery) (*user.User, error) {
-	user, err := s.store.GetByID(ctx, query.ID)
```

Comment:
so this is the meat of this PR right? removing the caseinsensitivecheck here

Reply:
Yep. The other things are just to not use this function inside auth code. It was never used anyway so

Reply:
Oh! Should the function be refactored to only take an in64? 

Looks like `CaseInsensitiveLoginConflict` would have been called with empty strings as far as I can see.

Reply:
That would make more sense when we only have one parameter any way
> Looks like CaseInsensitiveLoginConflict would have been called with empty strings as far as I can see.
No it was called with username / email from found user